tests/fakeprotocol_smart.py

```diff
@@ -658,8 +668,12 @@ async def _send_request(self, request_dict: dict):
             return self._set_on_off_gradually_info(info, params)
         elif method == "set_child_protection":
             return self._update_sysinfo_key(info, "child_protection", params["enable"])
-        # Vacuum special actions
-        elif method in ["playSelectAudio"]:
+        # Hub pairing
+        elif method in [
+            "begin_scanning_child_device",
+            "add_child_device_list",
+            "remove_child_device_list",
+        ] or method in ["playSelectAudio"]:
```

Comment:
```suggestion
        # actions
        elif method in [
            "begin_scanning_child_device",  #hub pairing
            "add_child_device_list",  #hub pairing
            "remove_child_device_list",  #hub pairing
            "playSelectAudio",  #vacuum special actions
        ]:
```
Perhaps this should be a tuple including required components so they only work if supported.

Reply:
Makes sense, but I'd rather fix that in a separate PR as it makes sense to do this for all existing methods.